tests/utils.js

```diff
@@ -42,8 +42,10 @@ let e2eDebug;
 const hasModal = () => element(by.css('#update-available')).isPresent();
 const COUCH_USER_ID_PREFIX = 'org.couchdb.user:';
 
-const COMPOSE_FILE = path.resolve(__dirname, 'cht-compose-test.yml');
-const COMPOSE_FILE_TEMPLATE = path.resolve(__dirname, '..', 'scripts', 'build', 'cht-compose.yml.template');
+const SERVICES_COMPOSE_FILE = path.resolve(__dirname, 'cht-services-test.yml');
+const COUCHDB_COMPOSE_FILE = path.resolve(__dirname, 'cht-couchdb-test.yml');
+const SERVICES_COMPOSE_FILE_TEMPLATE = path.resolve(__dirname, '..', 'scripts', 'build', 'cht-services.yml.template');
+const COUCHDB_COMPOSE_FILE_TEMPLATE = path.resolve(__dirname, '..', 'scripts', 'build', 'cht-couchdb.yml.template');
```

Comment:
NIT: This feels repetitive. Can we just have an array of filenames, eg:

```js
const COMPOSE_FILES = [ 'cht-services', 'cht-couchdb' ];
```

Then we can loop to generate the paths.

Reply:
Done.